app/controllers/admin/feedback_messages_controller.rb

```diff
@@ -107,7 +107,7 @@ def generate_message(params)
       <<~HEREDOC
         *New note from #{params['author_name']}:*
         *Report status: #{params['feedback_message_status']}*
-        Report page: https://#{ApplicationConfig['APP_DOMAIN']}/admin/reports/#{params['noteable_id']}
+        Report page: https://#{SiteConfig.app_domain}/admin/reports/#{params['noteable_id']}
```

Comment:
We could also use a path or url helper here, e.g. 

```ruby
admin_report_path(id: params['noteable_id'])
```

This seems like it would generally be a more robust way to generate internal links.

Reply:
I'm going to treat this as out of scope here just to cut off the little changes here and focus mostly on the core functional changes.